### PR TITLE
Upgrade to Hyrax 2.0.0

### LINKED ISSUE #425: Upgrade to Hyrax 2.0.0 final release
A quick attempt to upgrade fails with:
```
ActionView::Template::Error:                                                                                                                                                                                                                                               
        Missing partial hyrax/users/_notify_number with {:locale=>[:en], :formats=>[:html], :variants=>[], :handlers=>[:raw, :erb, :html, :builder, :ruby, :coffee, :jbuilder]}
```

### PR DESCRIPTION
**Pull request by little9**
This commit upgrades epigaea to Hyrax 2.0.0 

It includes some fixes for views, and updates to notifications.

Closes #425

### CODE REVIEW

File: app/views/_user_util_links.html.erb [hunk 1]
```diff
@@ -2,12 +2,12 @@
...
   <li>
-  
+    <%= render_notifications(user: current_user) %>
   </li>
+  <% if current_user.admin? %>
   <li>
-      <%= render 'hyrax/users/notify_number' %>
+
   </li>
   <% end %>
```
Comment (mark-dce) [inline app/views/_user_util_links.html.erb]: I'm not sure I understand what lines 8-12 are supposed to do, they just render as an empty 0px X 0px list item when I run the application.

**Comment by mark-dce** [comment]
@little9 - FYI - when I run this locally and try to create a new work, I'm getting a configuration error.  I'm not sure if there's some additional configuration setting I need somewhere.  Here's a gist of the error I got: https://gist.github.com/mark-dce/4eb9d7aac82e96ac61c3faf5e4d716f9

*STEPS TO REPRODUCE*
1. Login as an administrator
1. Go to your dashboard
1. Click on "Works" in the left navigation bar
1. Click on the "Add new work" button
1. Choose any work type and click on the "Create Work" button
1. Enter a title and display value
1. Attach a file
1. Click save to create a new work

*EXPECTED BEHAVIOR*
The system should create a new work

*ACTUAL BEHAVIOR*
```
ArgumentError in Hyrax::AudiosController#create
Missing host to link to! Please provide the :host parameter, set default_url_options[:host], or set :only_path to true
```
see full error and stack trace in gist linked above

**Comment by little9** [comment]
@mark-dce  You need to have values in the `.env.development` and I've been turning off the email features in my development environment, which can be done in `config/initializers/mailboxer.rb` by setting   `config.uses_emails = false`.

My `.env.developement` looks like this: 
```bash
export ACTION_MAILER_HOST=gmail.com
export RAILS_HOST=localhost:3000
export ACTION_MAILER_SMTP_ADDRESS=smtp.gmail.com
export ACTION_MAILER_PORT=465
export ACTION_MAILER_USER_NAME=???
export ACTION_MAILER_PASSWORD=??
```

File: app/views/_user_util_links.html.erb (hunk 1 above)
Comment (little9) [reply app/views/_user_util_links.html.erb]: @mark-dce That's the most technical description for an empty div I've ever read. 💯 

I removed it in a rebase, and the `admin` block. Non-admin users will need to a way to see notifications in the dashboard – right?

**Pull request merged**